.gitignore

```diff
@@ -172,3 +186,9 @@ nbdist/
 nbactions.xml
 nb-configuration.xml
 .nb-gradle/
+
+### Eclipse ###
```

Comment:
These should be blocked by [global gitignore](https://help.github.com/articles/ignoring-files/). Otherwise every project would in time contain all the possible IDE's temp files.